examples/test_context_based_restrictions_v1_examples.py

```diff
@@ -0,0 +1,417 @@
+# -*- coding: utf-8 -*-
+# (C) Copyright IBM Corp. 2021.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Examples for ContextBasedRestrictionsV1
+"""
+
+from ibm_cloud_sdk_core import ApiException, read_external_sources
+import os
+import pytest
+from ibm_platform_services.context_based_restrictions_v1 import *
+
+#
+# This file provides an example of how to use the Context Based Restrictions service.
+#
+# The following configuration properties are assumed to be defined:
+# CONTEXT_BASED_RESTRICTIONS_URL=<service base url>
+# CONTEXT_BASED_RESTRICTIONS_AUTH_TYPE=iam
+# CONTEXT_BASED_RESTRICTIONS_APIKEY=<IAM apikey>
+# CONTEXT_BASED_RESTRICTIONS_AUTH_URL=<IAM token service base URL - omit this if using the production environment>
+# CONTEXT_BASED_RESTRICTIONS_TEST_ACCOUNT_ID=<the id of the account under which test CBR zones and rules are created>
+# CONTEXT_BASED_RESTRICTIONS_TEST_SERVICE_NAME=<the name of the service to be associated with the test CBR rules>
+#
+# These configuration properties can be exported as environment variables, or stored
+# in a configuration file and then:
+# export IBM_CREDENTIALS_FILE=<name of configuration file>
+#
+config_file = 'context_based_restrictions_v1.env'
+
+context_based_restrictions_service = None
+
+config = None
+
+
+##############################################################################
+# Start of Examples for Service: ContextBasedRestrictionsV1
+##############################################################################
+# region
+class TestContextBasedRestrictionsV1Examples():
+    """
+    Example Test Class for ContextBasedRestrictionsV1
+    """
+
+    test_account_id = None
+    service_name = None
+    zone_id = None
+    zone_rev = None
+    rule_id = None
+    rule_rev = None
```

Comment:
Typically, in the working examples for other services (see test_case_management_v1_examples.py for an example), we just define these types of variables as globals above the class definition.   This would apply to all the variables in this block.   The advantage of doing it this way is that the variables can be referenced by merely their simple name with no need to include the classname.
```suggestion
```